Translations are fetched from upstream git repo at build time, so disable them.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,14 +30,14 @@
 # we need qpa/qplatformnativeinterface.h for global shortcut
 find_package(Qt5Gui REQUIRED)
 find_package(Qt5Widgets REQUIRED)
-find_package(Qt5LinguistTools REQUIRED)
+#find_package(Qt5LinguistTools REQUIRED)
 if(APPLE)
 elseif(UNIX)
     find_package(Qt5X11Extras REQUIRED)
 endif()
 find_package(QTermWidget5 REQUIRED)
-find_package(lxqt-build-tools ${LXQTBT_MINIMUM_VERSION} REQUIRED)
-include(LXQtTranslateTs)
+#find_package(lxqt-build-tools ${LXQTBT_MINIMUM_VERSION} REQUIRED)
+#include(LXQtTranslateTs)
 message(STATUS "Qt version: ${Qt5Core_VERSION_STRING}")
 
 include(${QTERMWIDGET_USE_FILE})
@@ -110,17 +110,17 @@
 qt5_wrap_ui( QTERM_UI ${QTERM_UI_SRC} )
 qt5_wrap_cpp( QTERM_MOC ${QTERM_MOC_SRC} )
 qt5_add_resources( QTERM_RCC ${QTERM_RCC_SRC} )
-lxqt_translate_ts(QTERM_QM
-    TRANSLATION_DIR "src/translations"
-    PULL_TRANSLATIONS
-        ${PULL_TRANSLATIONS}
-    CLEAN_TRANSLATIONS
-        ${CLEAN_TRANSLATIONS}
-    TRANSLATIONS_REPO
-        ${TRANSLATIONS_REPO}
-    TRANSLATIONS_REFSPEC
-        ${TRANSLATIONS_REFSPEC}
-)
+#lxqt_translate_ts(QTERM_QM
+#    TRANSLATION_DIR "src/translations"
+#    PULL_TRANSLATIONS
+#        ${PULL_TRANSLATIONS}
+#    CLEAN_TRANSLATIONS
+#        ${CLEAN_TRANSLATIONS}
+#    TRANSLATIONS_REPO
+#        ${TRANSLATIONS_REPO}
+#    TRANSLATIONS_REFSPEC
+#        ${TRANSLATIONS_REFSPEC}
+#)
 
 include_directories(
     "${CMAKE_SOURCE_DIR}"
